server/game/GameActions/PlaceFateAction.js

```diff
@@ -6,19 +6,27 @@ class PlaceFateAction extends CardGameAction {
     setDefaultProperties() {
         this.amount = 1;
         this.origin = null;
+        this.recipient = null;
```

Comment:
Using `this.recipient` here doesn't make any sense to me, because the target of the ability should be the recipient.  You don't actually use this parameter when generating the event which actually changes the game state